### ISSUE TITLE
problem with nano

### ISSUE THREAD

**Issue by jon-daemon** [paulkaplan/HexUploader#8]
I tried it with a nano compatible board (it uses a ch340 usb to serial chip).
I'm not able to upload the .hex file. I keep getting this message
```
avrdude: stk500v2_ReceiveMessage(): timeout
```
Is this a known issue? What could be the problem?

**Comment by Volchanick** [comment]
I get the same mistake.

**Comment by goto10** [comment]
Has anyone found a solution for this yet?

**Comment by mircobabini** [comment]
Same issue here :/

**Comment by nightflyer88** [comment]
The timeout means that AVRdude has no connection to the CPU. The problem is with the RS232 driver. I've had bad experiences with the CH340 chip, eg on MAC, the driver works badly. The FTDI chips work best.

**Comment by jon-daemon** [comment]
I also had troubles with ch340 in the past with third party drivers and kernel panics. The latest official driver has never given me an issue on High Sierra. I also use it on custom boards and it works fine. Also I think ch340 is natively supported on Mojave and Catalina but I haven't tried it myself.
The serial port is visible on HexUploader. It just gives that timeout error.

**Comment by mircobabini** [comment]
That's right, same here. CH340 driver works perfectly with Arduino software and lot of nanos from different manufacturers. HexUploader sees the serial port, but gives that timeout errore.

I will use windows :/
